showkase-processor/src/main/java/com/airbnb/android/showkase/processor/writer/PaparazziShowkaseScreenshotTestWriter.kt

```diff
@@ -22,7 +22,7 @@ class PaparazziShowkaseScreenshotTestWriter(private val environment: XProcessing
         rootModulePackageName: String,
         testClassName: String
     ) {
-        val showkaseScreenshotTestClassName = "${testClassName}_PaparazziShowkaseTest"
+        val showkaseScreenshotTestClassName = "${testClassName}Impl"
```

Comment:
Note: This contains the fixes in https://github.com/airbnb/Showkase/pull/419 to shorten the generated Paparazzi image filenames - due to changes in Paparazzi, we can now encounter `java.io.FileNotFoundException: File name too long` on some systems.